src/ecb.rs

```diff
@@ -0,0 +1,282 @@
+use chrono;
+use roxmltree;
+
+pub fn get_eur_to_usd_exchange_rate(
```

Comment:
why this is named "get_eur_to_usd_exchange_rate" ? The exchange rate we need is USD to EUR ?

Reply:
My understanding is that we need USD to EUR because I assume that the denominator is 1 not nominator. In case of  USD/EUR (which I read as usd to euro) if denominator is 1 than USD = 1 and EUR < 1 and overall value is < 1.

in this function I download the opposite EUR/USD and invert it because I haven't seen the inverse anywhere on ECB api.

Reply:
I can change it if you prefer, I'm not fluent in financial market terminology